Zype/Const.swift

```diff
@@ -38,7 +38,8 @@ class Const: NSObject {
     
     // MARK: - Feature Flags
     
-    static let kNativeSubscriptionEnabled = false
+    static let kNativeSubscriptionEnabled = true
```

Comment:
@advantiss425 let's make sure these feature flags remain = false by default (right now this PR turns them to 'true' by default for all customers)

Reply:
@imchrissmith can you merge this PR first?  because I removed the repository for this PR.
After you merge this PR, then I'll update the flags at once.